### DISCUSSION TITLE
Weird Bug ?

### DISCUSSION THREAD

**Discussion by Kenpachi-420** [SideStore/SideStore#455] [Q&A] [answered]
Sorry been having an issue with sidestore i cant quite figure out. When I was using altserver to install the sidestore ipa i never had any issues on refresh. After switching over and using sideserver to install/update sidestore there has been a problem refreshing. It will install/refresh new apps no problem, however it will not refresh sidestore. Acts like it does and closes and all but open back up with same amount of days. Tried to troubleshoot myself and have search git and google for similar issues with no luck. The last thing i tried was removing sidestore all all apps from my phone and starting all over. Tried that with sidestore auto installing ipa and even using shift click to install nightly ipa. Both allow all other apps to refresh but not sidestore. Also, as soon as using sideserver to install ipa i disconnect phone from pc, make sure phone on wifi and wireguard vpn and immediatly refresh sidestore. Again it looks like it works. However next day when it shows 6 days and i try again, it stays on 6 days. No errors of any kinda. On an iphone xr with 15.6, doing same on an iphone xr with 16.5. Both previously worked(with altserver install). Only option i feel i havent tried was doing back to try altserver again. Any help appreciated.
Thanks

**Comment by Kenpachi-420** [comment]
Anyone have this happen to them? or have an idea? I am still having same issue. Have to install sidestore with pc every couple day. Wifi refresh works on other apps sidestore installs, but not sidestore.

**Comment by Kenpachi-420** [comment]
After going back and using AltStore/paring file everything seems to work like usual again. I am guessing that means for some reason the sideserver doesnt pair properly for me. I am using this on 2 seperate iphones but same PC. Maybe thats a reason? Any idea please let me know. I would prefer to use your sideserver if possible.

**Comment by Kenpachi-420** [comment]
well.. thanks for all the information. its been super not helpful...

**Comment by Spidy123222** [comment, marked as answer]
Probably should of looked in issues as this is already a reported issue and this is in discussion which isn't really a place to report issues. https://github.com/SideStore/SideStore/issues/450
